vehicle/OVMS.V3/components/vehicle_nissanleaf/src/vehicle_nissanleaf.cpp

```diff
@@ -1264,6 +1264,11 @@ void OvmsVehicleNissanLeaf::IncomingFrameCan1(CAN_frame_t* p_frame)
         {
         StandardMetrics.ms_v_bat_temp->SetValue(d[2] / 2 - 40);
         }
+      // Battery Heater existance as reported by the LBC. Frame 4, bit 0.
```

Comment:
Keep in mind this only works on AZE0 not ZE0, it will say 0 on the ZE0 even though heater is present!

Reply:
Ah right, fixed that so that it does not show up positive on Leafs prior to 2013. Will compile, test and add to this pull rq.

Reply:
fixed